libs/sdk-model/src/execution/attribute/factory.ts

```diff
@@ -73,6 +73,38 @@ export class AttributeBuilder {
         return this;
     };
 
+    /**
+     * Sets show all values property.
+     *
+     * @remarks
+     * The flag showAllValues translates to a property of the same name on the attribute in execution definition.
+     * If truthy, the backend will return all values of the particular attribute in the execution response
+     * even if there are no data available for it.
+     *
+     * @param showAllValues - flag defining whether to return all attribute values for given attribute; undefined to use backend default behavior(false)
+     */
+    public showAllValues = (showAllValues?: boolean | undefined): this => {
+        if (showAllValues === undefined) {
+            return this.undefinedShowAllValues();
+        }
+
+        this.attribute.showAllValues = showAllValues;
+
+        return this;
+    };
+
+    /**
+     * Resets show all values property.
+     *
+     * @remarks
+     * The backend default behavior will be used - not showing all values
+     */
+    public undefinedShowAllValues = (): this => {
```

Comment:
This naming seems a bit strange + I'm not sure whether it has a big added value, as it's just an alias for `. showAllValues()` call

Reply:
removed